samples/05_content_publishers/pdf_table_to_pdf_map.ipynb

```diff
@@ -1246,8 +1246,7 @@
    ],
```

Comment:
<p>reference 3 needs to be updated to the new doc :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>